back-end/api/quickstart/signals.py

```diff
@@ -7,6 +7,11 @@
 # This will auto-create an author when needed
 @receiver(user_signed_up)
 def after_user_signed_up(request, user, **kwargs):
-    #TODO: Make this url a uuid
-    url = user.id
+    """
+    After a user has signed up, an author is automatically
+    created with the appropriate id containing a UUID
+    Get absolute URI: https://stackoverflow.com/a/10119243
+    """
+    hex_uuid = str(uuid.uuid4().hex)
+    url = str(request.build_absolute_uri('/')) + "author/" + hex_uuid
```

Comment:
Just want to confirm that this url contains the port of the server (and not the client) like Chris was concerned about.

Reply:
yes, from what I tested the urls returned are `http://localhost:8000/author/author_id`